cmake/windows-setup.cmake

```diff
@@ -3,114 +3,113 @@
-set(FLB_OUT_RETRY              No)
-set(FLB_OUT_SPLUNK            Yes)
-set(FLB_OUT_STACKDRIVER       Yes)
-set(FLB_OUT_STDOUT            Yes)
-set(FLB_OUT_LIB                No)
-set(FLB_OUT_NULL              Yes)
-set(FLB_OUT_FLOWCOUNTER       Yes)
-set(FLB_OUT_KAFKA              No)
-set(FLB_OUT_KAFKA_REST         No)
-set(FLB_OUT_CLOUDWATCH_LOGS   Yes)
-set(FLB_OUT_S3                Yes)
-set(FLB_OUT_KINESIS_FIREHOSE   No)
+  # OUTPUT plugins
+  # ==============
+  set(FLB_OUT_AZURE             Yes)
+  set(FLB_OUT_AZURE_BLOB        Yes)
+  set(FLB_OUT_BIGQUERY           No)
+  set(FLB_OUT_COUNTER           Yes)
+  set(FLB_OUT_DATADOG           Yes)
+  set(FLB_OUT_ES                Yes)
+  set(FLB_OUT_EXIT               No)
+  set(FLB_OUT_FORWARD           Yes)
+  set(FLB_OUT_GELF              Yes)
+  set(FLB_OUT_HTTP              Yes)
+  set(FLB_OUT_INFLUXDB          Yes)
+  set(FLB_OUT_NATS               No)
+  set(FLB_OUT_PLOT               No)
+  set(FLB_OUT_FILE              Yes)
+  set(FLB_OUT_TD                 No)
+  set(FLB_OUT_RETRY              No)
+  set(FLB_OUT_SPLUNK            Yes)
+  set(FLB_OUT_STACKDRIVER       Yes)
+  set(FLB_OUT_STDOUT            Yes)
+  set(FLB_OUT_LIB               Yes)
+  set(FLB_OUT_NULL              Yes)
+  set(FLB_OUT_FLOWCOUNTER       Yes)
+  set(FLB_OUT_KAFKA              No)
+  set(FLB_OUT_KAFKA_REST         No)
+  set(FLB_OUT_CLOUDWATCH_LOGS   Yes)
+  set(FLB_OUT_S3                Yes)
+  set(FLB_OUT_KINESIS_FIREHOSE   No)
 
-# FILTER plugins
-# ==============
-set(FLB_FILTER_GREP           Yes)
-set(FLB_FILTER_MODIFY         Yes)
-set(FLB_FILTER_STDOUT         Yes)
-set(FLB_FILTER_PARSER         Yes)
-set(FLB_FILTER_KUBERNETES     Yes)
-set(FLB_FILTER_THROTTLE       Yes)
-set(FLB_FILTER_THROTTLE_SIZE  Yes)
-set(FLB_FILTER_NEST           Yes)
-set(FLB_FILTER_LUA            Yes)
-set(FLB_FILTER_RECORD_MODIFIER Yes)
-set(FLB_FILTER_REWRITE_TAG    Yes)
-set(FLB_FILTER_GEOIP2         Yes)
+  # FILTER plugins
+  # ==============
+  set(FLB_FILTER_GREP           Yes)
+  set(FLB_FILTER_MODIFY         Yes)
+  set(FLB_FILTER_STDOUT         Yes)
+  set(FLB_FILTER_PARSER         Yes)
+  set(FLB_FILTER_KUBERNETES     Yes)
+  set(FLB_FILTER_THROTTLE       Yes)
+  set(FLB_FILTER_THROTTLE_SIZE  Yes)
+  set(FLB_FILTER_NEST           Yes)
+  set(FLB_FILTER_LUA            Yes)
+  set(FLB_FILTER_RECORD_MODIFIER Yes)
+  set(FLB_FILTER_REWRITE_TAG    Yes)
+  set(FLB_FILTER_GEOIP2         Yes)
+endif()
 
 # Search bison and flex executables
-if(CMAKE_SYSTEM_NAME MATCHES "Windows")
-  find_package(FLEX)
-  find_package(BISON)
+find_package(FLEX)
+find_package(BISON)
 
-  if (NOT (${FLEX_FOUND} AND ${BISON_FOUND}))
-    message(STATUS "flex and bison not found. Disable stream_processor building.")
-    set(FLB_STREAM_PROCESSOR No)
-    set(FLB_RECORD_ACCESSOR  No)
-  endif()
+if (NOT (${FLEX_FOUND} AND ${BISON_FOUND}))
+  message(STATUS "flex and bison not found. Disable stream_processor building.")
```

Comment:
Just a nit about a preexisting issue, it may be good to separate the error messages in 2 different conditionals so if someone is trying to build it and they're missing one of them they can properly address it. When I got started with fluent-bit on windows I found the required flex and bison versions to be a pain point.